packages/shared/src/modules/auth/libs/exceptions/auth-error/auth-error.exception.ts

```diff
@@ -1,11 +1,12 @@
-import { HTTPError } from "../../../../../libs/exceptions/exceptions.js";
-import { HTTPCode } from "../../../../../libs/modules/http/http.js";
+import { HTTPError } from "../../../../../libs/exceptions/http-error/http-error.exception.js";
+import { HTTPCode } from "../../../../../libs/modules/http/libs/enums/http-code.enum.js";
+import { ValueOf } from "../../../../../libs/types/value-of.type.js";
```

Comment:
```suggestion
import { HTTPError, HTTPCode } from "../../../../../libs/modules/http.js";
import { ValueOf } from "../../../../../libs/types/types.js";
```
HTTPError should be a part of http module